media-api/app/lib/elasticsearch/impls/elasticsearch1/SearchFilters.scala

```diff
@@ -60,15 +60,14 @@ class SearchFilters(config: MediaApiConfig) extends ImageFields {
   )
 
   val persistedFilter: FilterBuilder = filters.or(
-    filters.bool.must(filters.existsOrMissing("exports", exists = true)),
-    filters.bool.must(filters.existsOrMissing("usages", exists = true)),
-    filters.exists(NonEmptyList(identifierField(config.persistenceIdentifier))),
-    filters.bool.must(filters.boolTerm(editsField("archived"), value = true)),
-    filters.exists(NonEmptyList(editsField("metadata"))),
-    filters.bool.must(filters.terms(usageRightsField("category"), persistedCategories)),
-    filters.bool.must(filters.terms(collectionsField("path"), config.persistedRootCollections.toNel.get)),
-    filters.exists(NonEmptyList(editsField("photoshoot")))
-  )
+    filters.bool.must(filters.existsOrMissing("exports", exists = true)), // has been cropped
+    filters.bool.must(filters.existsOrMissing("usages", exists = true)), // used in content
+    filters.exists(NonEmptyList(identifierField(config.persistenceIdentifier))), // pre-Grid content
+    filters.bool.must(filters.boolTerm(editsField("archived"), value = true)), // added to Library
+    filters.exists(NonEmptyList(editsField("metadata"))), // user changes to image metadata
+    filters.bool.must(filters.terms(usageRightsField("category"), persistedCategories)), // has specific Usage Rights
+    filters.bool.must(filters.terms(collectionsField("path"), config.persistedRootCollections.toNel.get)), // added to GNM Archive or persisted Collections
+    filters.exists(NonEmptyList(editsField("photoshoot"))) // added to any Photoshoot
```

Comment:
I think there might be a missing paren here?

Reply:
Good catch. This is fixed and will merge now.